config/crd/bases/crd-druid.gardener.cloud_etcds.yaml

```diff
@@ -0,0 +1,2106 @@
+---
```

Comment:
CRDs are no longer present in `config/crd/bases` - please ensure that these are removed